universal-application-tool-0.0.1/app/views/components/Modal.java

```diff
@@ -60,7 +72,9 @@ public static ModalBuilder builder(String modalId, Tag content) {
 
     // Optional fields. See #setOptionalFields().
     private String modalTitle;
-    private String buttonText;
+    private String triggerButtonText;
+
+    private Tag triggerButtonContent = null;
```

Comment:
Suggestion (blocking):
```suggestion
    private Optional<Tag> triggerButtonContent = Optional.empty();
```

Reply:
good call